python/cudf/cudf/_lib/pylibcudf/gpumemoryview.pxd

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+
+cdef class gpumemoryview:
```

Comment:
Is there any harm in using this name? It signals a clear intent, and since it's within pylibcudf it's unlikely that anyone will think of it as a generally usable utility. Keeping the same name will save some refactoring work if/when we do get to having a complete proposal.

Reply:
Nope - I think it's fine